docs/devices/E1746.md

```diff
@@ -26,6 +26,10 @@ While pairing the LED is flashing/dimming slowly. Once the pairing is finished,
 ## OTA updates
 This device supports OTA updates, for more information see [OTA updates](../information/ota_updates.md).
 
+## Keep alive payload
+The device send a payload exactly 1 hours after the first pairing and a payload each exactly 4 hours after.  
```

Comment:
By just reading this text, it's unclear to me what todo/expect from it.

Reply:
This device send one empty payload each 4 hours. If your last seen is more than 4 hours, so the device stop to work.
It's very usefull to monitoring devices that stop to work.

Reply:
Then please add this info